src/integrations/prefect-dask/prefect_dask/client.py

```diff
@@ -11,7 +11,7 @@
 from prefect.utilities.engine import collect_task_run_inputs_sync
 
 
-class PrefectDistributedClient(Client):
+class PrefectDaskClient(Client):
```

Comment:
I also renamed this client to be more consistent with the `DaskTaskRunner` and `PrefectDaskFuture`.